modules/ROOT/pages/logs.adoc

```diff
@@ -29,31 +26,32 @@ Do not override the default JVM timezone for applications deployed to CloudHub a
 
 ====
 
+[[log-search]]
 == Search Logs
 
-In Anypoint Monitoring, click *Log Search* in the menu on the left. On the log search page, you can search for text in logs in a few different ways. You can type your search query into the search field, you can use the UI to add a filter, and you can create a query.
-
-When you search for logs for APIs deployed through Flex Gateway, use the *Application* filter. Use the *App ID* field to identify the instance.
-
-=== Log Search Query
+From Anypoint Monitoring, find *Log Search* to search for text in logs from the search field, through a filter, or with a DSL query.
 
-To search the logs, type your search query into the log search field and press *Enter*.
+image::intro-logs.png["Logs in Anypoint Monitoring"]
 
-Provide the exact term you're looking for, such as `"code 400"` to find  bad requests. For
-exact matches of more than one word, surround the search term
-with quotation marks.
+* <<search-field>>
+* <<search-filter>>
+* <<search-dsl>>
 
-Example results from a `"code 400"` search:
-
-image::log-search-string.png[Log Search]
+When you search for logs for APIs deployed through Flex Gateway, use the *Application* filter. Use the *App ID* field to identify the instance.
 
-Use the asterisk wildcard in a search if you don't know the exact term.
+[[search-field]]
+=== Search from the Log Search Field
 
-Example results from the wildcard search `payload=org.glassfish.grizzly*`:
+To search the logs, type your search query into the log search field and press *Enter*. 
 
-image::log-search-wildcard.png[Log Search Using a Wildcard]
+* Search for a term, such as `"Info Dynamic Logging"`.
```

Comment:
I think, in this case, just the "" is necessary (no font change).

Reply:
```suggestion
* Search for a term, such as "Info Dynamic Logging".
```

Reply:
correcting

Reply:
ok, based on comment below